test/e2e/commands/dual-cluster-full.test.ts

```diff
@@ -69,7 +69,6 @@ const endToEndTestSuite: EndToEndTestSuite = new EndToEndTestSuiteBuilder()
       ClusterReferenceTest.connect(options);
       DeploymentTest.create(options);
       DeploymentTest.addCluster(options);
-      ClusterReferenceTest.setup(options);
       NodeTest.keys(options);
```

Comment:
no need to call `cluster-ref config setup` since already called in `setup-dual-e2e.sh`